torch_sim/models/fairchem.py

```diff
@@ -257,11 +277,9 @@ def __init__(  # noqa: C901, PLR0915
             inference_only=True,
         )
 
-        self.trainer.model = self.trainer.model.eval()
-
         if dtype is not None:
             # Convert model parameters to specified dtype
-            self.trainer.model = self.trainer.model.to(dtype=self.dtype)
+            self.trainer.model = self.trainer.model.to(dtype=self._dtype)
```

Comment:
this is a property of the ModelInterface and so it's better to access that rather than the underlying "private" attribute